bin/util.js

```diff
@@ -205,10 +205,13 @@ let printProgress = ({ percent, total, transferred }) => {
       line += ` of ${roundedTotalMbs} MB`;
     }
   }
-
-  process.stdout.clearLine();
-  process.stdout.cursorTo(0);
-  process.stdout.write(line);
+  try {
+      process.stdout.clearLine();
+      process.stdout.cursorTo(0);
+      process.stdout.write(line);
+  } catch (e) {
+      console.debug(e);
```

Comment:
Maybe we should drop that line, when thinking about it... It would just clutter the output.